00-HTML-CSS-basics/layout.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta content="name=value,name=value" name="viewport">
+    <meta charset="utf-8">
+    <link href="layout.css" rel="stylesheet" type="text/css">
+    <title>Layout</title>
+</head>
+<body>
+
+<!-- PROBLEM: when the mouse pointer in hover the text, the image stays in black and white -->
```

Comment:
Code-wise this file is pretty well done. I will take a deeper look into it ASAP.